lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -1713,17 +1713,10 @@ export class StandardController implements IController {
     }
 
     /**
-     * Returns the native internal storage
+     * Sets interaction in progress temp cache flag
      */
-    public getNativeInternalStorage(): BrowserCacheManager {
-        return this.nativeInternalStorage;
-    }
-
-    /**
-     * Returns the instance of interface for crypto functions
-     */
-    public getBrowserCrypto(): ICrypto {
-        return this.browserCrypto;
+    public setInteractionInProgress(progress: boolean): void {
```

Comment:
Public access for 1p.

Reply:
1P shouldn't need to set this, if it does we should see what we need to do to make it not needed

Reply:
I would like that in a separate PR though. @konstantin-msft  please correct me but my assumption is that this is reflecting current 1P status?

Reply:
Removed the API after brief discussion.